src/internal/util/identity.ts

```diff
@@ -1,3 +1,29 @@
+/**
+ * This function takes one parameter and just returns it. Simply put,
+ * this is like `<T>(x: T): T => x`.
+ *
+ * ## Example
+ *
+ * This is useful in some cases when using things like `mergeMap`
+ *
+ * ```ts
+ * import { interval, take, map, range, mergeMap, identity } from 'rxjs';
+ *
+ * const source$ = interval(1000).pipe(take(5));
+ *
+ * const result$ = source$.pipe(
+ *   map(i => range(i)),
+ *   mergeMap(identity) // same as mergeMap(x => x)
+ * );
+ *
+ * result$.subscribe({
+ *   next: console.log
+ * });
+ * ```
+ *
```

Comment:
One of the main use cases I have for this is with a ternary to selectively apply an operator:

```ts
source$.pipe(
  condition ? mergeMap(doSomething) : identity
)
```

It would be nice to have an example of that.

Reply:
I'll add.

Reply:
Added (thought, as the second example). If you'd like this example to be the first one, I can swap them.